pinchy/src/events.rs

```diff
@@ -273,6 +273,99 @@ pub async fn handle_event(event: &SyscallEvent, formatter: Formatter<'_>) -> any
 
             finish!(sf, event.return_value);
         }
+        syscalls::SYS_setxattr => {
+            let data = unsafe { event.data.setxattr };
+            argf!(sf, "pathname: {}", format_path(&data.pathname, false));
+            argf!(sf, "name: {}", format_path(&data.name, false));
+            argf!(sf, "value: {}", format_bytes(&data.value[..data.size.min(data.value.len())]));
+            argf!(sf, "size: {}", data.size);
+            argf!(sf, "flags: {}", data.flags);
```

Comment:
The flags should be parsed, check the instructions for details, adjust tests after fixing.